apps.awesim.org/apps/myjobs/templates/Basic_R_Parallel_Job_Owens/manifest.yml

```diff
@@ -0,0 +1,6 @@
+name: Basic R Parallel Job - Owens
+host: owens
+script: pbs_script.sh
+notes: |
+  A Basic R Parallel Job for the OSC Owens Cluster
```

Comment:
The notes for this job should include that it is a comparison of several different methods for performing parallel computation in R.